.github/workflows/github.yml

```diff
@@ -151,6 +151,35 @@ jobs:
       if: ${{ startsWith(matrix.platform, 'msvc') }}
       run: source '${{github.workspace}}/CI/install_vcpkg_dependencies.sh' '${{matrix.platform}}'
 
+    - name: Setup Ninja (Windows)
+      if: ${{ startsWith(matrix.platform, 'msvc') }}
+      uses: seanmiddleditch/gha-setup-ninja@v5
```

Comment:
Runners already have ninja installed, why is this needed?

Reply:
Not needed, removed